migrations/versions/cd83b3e2832e_add_locked_state_to_apps.py

```diff
@@ -0,0 +1,26 @@
+"""
```

Comment:
don't suppose you'll want to split this into 2 PR's? or maybe 3?

db migrations
BE py changes
FE JS changes
?

Reply:
it doesn't really matter for freight, it's all just one docker image